bin/test_integration_rhel

```diff
@@ -1,23 +1,59 @@
 #!/bin/bash -ex
 
+function print_help() {
+  cat << EOF
+
+Run integration tests on RHEL container
+
+Usage: test_integration_rhel [options]
+
+    -h, --help            Shows this help message.
+
+    --rhel-version=7      Used to Set the RHEL version to test (e.g --rhel-version=7). Default version is 8.
```

Comment:
i would change `--rhel-version=7` to `--rhel-version` or `--rhel-version=<version>`. whichever you think is better. The example is already given

Reply:
also change `Set` to `set`